test-requirements.txt

```diff
@@ -16,7 +16,7 @@ pytest-instafail~=0.3.0
 pytest-mock~=1.1
 pytest-profiling~=1.3.0
 pytest-reorder~=0.1.0
-git+https://github.com/jayvdb/pytest-reqs@skip-vcs-reqs#egg=pytest-reqs
```

Comment:
We have another MR in the mobans repo which will finally solve this for all repos.

Reply:
This MR has been merged